### PR TITLE
kRO Zero Update

### PR DESCRIPTION
**Pull request by alisonrag**
* Update in Servertype files
* Update in table files
* Added some new maps

### CODE REVIEW

File: src/Network/Send/Zero.pm [hunk 1]
```diff
@@ -22,16 +22,18 @@ sub new {
 	my ($class) = @_;
 	my $self = $class->SUPER::new(@_);
 
-	my %packets = (		
-		'0ACF' => ['master_login', 'a4 Z25 a32 a5', [qw(game_code username password_rijndael flag)]],
+	my %packets = (
+		'0439' => ['item_use', 'a2 a4', [qw(ID targetID)]],
```
Comment (sctnightcore) [inline src/Network/Send/Zero.pm]: packet item_use is 0439 ?

Comment (alisonrag) [reply src/Network/Send/Zero.pm]: need to test, but as someone sayd in discord can be 0439

Comment (alisonrag) [reply src/Network/Send/Zero.pm]: its working in 0439

**Comment by comwhat** [comment]
How to edited clientinfo...?

**Pull request merged**